Source/Filter Effects/feGaussianBlur/SvgGaussianBlur.cs

```diff
@@ -222,23 +241,12 @@ public Bitmap Apply(Image inputImage)
         /// Gets or sets the radius of the blur (only allows for one value - not the two specified in the SVG Spec)
         /// </summary>
         [SvgAttribute("stdDeviation")]
-        public float StdDeviation
+        public SvgNumberCollection StdDeviation
         {
-            get { return _stdDeviation; }
-            set
-            {
-                if (value <= 0f)
-                {
-                    throw new InvalidOperationException("Radius must be greater then 0");
-                }
-                _stdDeviation = value;
-                Attributes["stdDeviation"] = value;
-                PreCalculate();
-            }
+            get { return GetAttribute("stdDeviation", false, new SvgNumberCollection() { 0f, 0f }); }

```

Comment:
https://www.w3.org/TR/SVG11/filters.html#feGaussianBlurStdDeviationAttribute
`If the attribute is not specified, then the effect is as if a value of 0 were specified.`

So, 
```cs
get { return GetAttribute("stdDeviation", false, new SvgNumberCollection() { 0f }); }
```

Reply:
If only one value is set than second is set to first value. It is expected to have two values.

Reply:
Third argument of GetAttribute method is default value if not specified.
So, please match with specification.

The following are recognized as equivalent:

```cs
get { return GetAttribute("stdDeviation", false, new SvgNumberCollection() { 0f }); }
```

```cs
get { return GetAttribute("stdDeviation", false, new SvgNumberCollection() { 0f, 0f }); }
```

Reply:
Pushed changes to PR